server/endpoints/models/User.js

```diff
@@ -0,0 +1,48 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const UserSchema = new Schema({
+  username: {
+    type: String,
+    unique: true,
+    required: true
+  },
+  publicName: {
+    type: String,
+    required: true
+  },
+  language: {
+    type: String,
+    required: true
+  },
+  rating: {
```

Comment:
i think we need another thing here, like positiveRatingCount and negativeRatingCount or something.

Reply:
changed